packages/palette/src/elements/Button/Button.tsx

```diff
@@ -184,22 +184,31 @@ export class ButtonBase extends Component<ButtonBaseProps & SansProps> {
       >
         {loading && <Spinner size={this.props.buttonSize} />}
 
-        <VisibleText
-          pt="1px"
-          weight={weight || "medium"}
-          color={color}
-          size={size}
-        >
-          {children}
-        </VisibleText>
-        <HiddenText
-          role="presentation"
-          pt="1px"
-          weight={weight || "medium"}
-          size={size}
-        >
-          {longestText ? longestText : children}
-        </HiddenText>
+        {longestText && (
+          <>
+            <VisibleText
+              pt="1px"
+              weight={weight || "medium"}
+              color={color}
+              size={size}
+            >
+              {children}
+            </VisibleText>
+            <HiddenText
+              role="presentation"
+              pt="1px"
+              weight={weight || "medium"}
+              size={size}
+            >
+              {longestText}
+            </HiddenText>
+          </>
+        )}
+        {!longestText && (
+          <Sans pt="1px" weight={weight || "medium"} size={size}>
+            {children}
+          </Sans>
+        )}
```

Comment:
maybe 
```js
{longestText 
  ? ...
  : ...
}
```

Reply:
Yes, just switched to the ternary, it is cleaner.